lib/manager/gradle-lite/readme.md

```diff
@@ -0,0 +1 @@
+This is an alternate manager for Gradle, written completely in JS.
```

Comment:
```suggestion
This is an alternate manager for Gradle, written completely in JavaScript.
```

If this is about JavaScript, I think it's clearer to expand JS -> JavaScript. That way it's clear straightaway what language it uses.